config/default..sample-with-domain.json

```diff
@@ -0,0 +1,34 @@
+{
+  "app": {
+    "prefix": "nf",
+    "name": "s3-webhosting"
+  },
+  "domain": {
+    "hostedZoneId": "HOSTED-ZONE-ID",
```

Comment:
Is this meant to be replaced?  Out of scope for this PR but I think w YAML we can do inline comments for items like this.

Reply:
Yes, these values are placeholders only and must be replaced with valid values.